zipline/data/minute_bars.py

```diff
@@ -441,6 +441,13 @@ def pad(self, sid, date):
         assert new_last_date == date, "new_last_date={0} != date={1}".format(
             new_last_date, date)
 
+    def set_bcolz_attrs(self, sid, **kwargs):
```

Comment:
This is in support of adding the tick size for each future?

I would recommend taking `bcolz` out of the name, instead something like `set_sid_attrs` or `set_sid_metadata` so that the public writer interface is not tied to implementation.

We should add a test covering this method.